bestiary/core/schema.py

```diff
@@ -120,8 +122,15 @@ class Meta:
         model = Ecosystem
 
 
+class ProjectType(DjangoObjectType):
+    """Project objects are meant to gather a set of end-point views"""
```

Comment:
The description of a project is not very clear. To talk about end-point views is confusing. We can leave it this way for now but we will probably change it in the future.

Reply:
From the comment where I posted another definition, this could updated to:

`Project objects are meant to group a set of data locations`. 

WDYT?

Reply:
That looks better.